index.js

```diff
@@ -140,9 +140,9 @@ const parseMMD = /*#__PURE__*/function () {
   };
 }();
 
-module.exports = function mermaidParse(definition, config) {
+module.exports = function mermaidParse(definition, config, puppeteerConfig = null) {
   return _asyncToGenerator(function* () {
-    const browser = yield puppeteer.launch();
+    const browser = puppetterConfig ? yield puppeteer.launch(puppeteerConfig) : yield puppeteer.launch();
```

Comment:
I believe if you initialize `puppeteerConfig` with `undefined` you can just pass it it and it will be ignored if no config is passed
```suggestion
    const browser = yield puppeteer.launch(puppeteerConfig);
```

Reply:
I am not sure about this possibility, so I went with the ternary.